<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="accountTab">
       <attribute name="title">
        <string>Account</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <widget class="QTableWidget" name="pendingOrdersTable"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="activeLabel">
          <property name="text">
           <string>Active</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="pendingLabel">
          <property name="text">
           <string>Pending</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTableWidget" name="activePositionsTable">
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked</set>
          </property>
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="accountValueFrame">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="netLiquidationValueLabel">
             <property name="text">
              <string>Net Liq. Value :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="currentAvailableFundsLabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="cashBalanceLabel">
             <property name="text">
              <string>Cash Balance :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="dayTradesRemainingLabel">
             <property name="text">
              <string>Day Trades Remaining :</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="strategiesTab">
       <attribute name="title">
        <string>Strategies</string>
       </attribute>
       <widget class="QComboBox" name="strategiesListCombo">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EarningsVolatilityStrat</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>50</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1148</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
